### PR TITLE
Update NodeSource's distributions landing page

### PR DESCRIPTION
**Pull request by julianduque**
We added a new landing page for our Linux Distributions, here is the update, thanks!

### CODE REVIEW

**Comment by ghost** [comment]
[Travis build passed](https://travis-ci.org/nodejs/nodejs.org/builds/120068511) :+1:

**Comment by fhemberger** [comment]
A newsletter sign-up form, seriously? Behind a link shortener?

**Comment by hg** [comment]
so where exactly CAN you access the information?

**Comment by Fishrock123** [comment]
pretty sure that is not the correct link?

**Comment by Fishrock123** [comment]
@julianduque Do we have https for the shortener? (Though really, a full link would probably be preferred.)

**Comment by fhemberger** [comment]
@Fishrock123 https://content.nodesource.com/welcome would work as well, and yes, we should definitely go without the shortener.

Still not convinced this is the best target for the link.

**Comment by lpinca** [comment]
Is this NodeSource April Fools joke?

**Comment by julianduque** [comment]
@Fishrock123 I can update the link to https://content.nodesource.com/welcome  and yes, it is the correct link

File: locale/en/download/package-manager.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ pacman -S nodejs npm
 
 Also including: **Linux Mint**, **Linux Mint Debian Edition (LMDE)**, **elementaryOS** and others.
 
-Node.js is available from the [NodeSource](https://nodesource.com) Debian and Ubuntu binary distributions repository (formerly [Chris Lea's](https://github.com/chrislea) Launchpad PPA). Support for this repository, along with its scripts, can be found on GitHub at [nodesource/distributions](https://github.com/nodesource/distributions).
+Node.js is available from the [NodeSource](http://nsrc.io/release-updates-2) Debian and Ubuntu binary distributions repository (formerly [Chris Lea's](https://github.com/chrislea) Launchpad PPA). Support for this repository, along with its scripts, can be found on GitHub at [nodesource/distributions](https://github.com/nodesource/distributions).
```
Comment (Fishrock123) [inline locale/en/download/package-manager.md:39]: How about if we keep the existing link to nodesource and add after that you can sign up to get immediate updates when new versions come out? Does that work?

**Comment by julianduque** [comment]
Thanks for the feedback, we are going to use another approach, this PR is no longer needed :)

**Comment by lpinca** [comment]
It was no joke, I thought it was, sorry @julianduque.

**Comment by julianduque** [comment]
@lpinca not a joke at all, we are trying to give more information about our distributions for people clicking on the link and give them the option to subscribe to updated, so we might be rewording the copy

**Pull request closed without merging**